packages/common/concurrency.js

```diff
@@ -78,60 +78,7 @@ const promiseUrl = (urlstr) =>
     }).on('error', reject);
   });
 
-class Mutex {
```

Comment:
This code was both unused/referenced.   Removed to lock script.

Reply:
This probably should require a CHANGELOG/deprecation.... but I doubt it's in use by anyone actively using the last few versions - I'm 50/50 on just making it a breaking change notice and removing it as a requirement.   It's an easy change either way.

Reply:
See notes above on the new location of this class